client/src/components/game/lab2/components/game.js

```diff
@@ -0,0 +1,132 @@
+import React, {Component} from 'react';
+import Countdown from 'react-countdown-now';
+import SecondTimer from './secondTimer';
+import Instructions from '../Instructions/instructions';
+import Circle from './circle';
+import Title from './header/title';
+import './gameStyle.css';
+
+/*
+Class for controlling the main game page
+*/
+class game extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      startTime: 0
+    }
+  }
+
+  //custom renderer for the class
+  render() {
+    const {correctColor, incorrectColorOne, incorrectColorTwo,
+      background, gameOption, resetOption, onChangeGameColors, colors,
+      resetColors, selectOption, gameEnded, enterInfoState, enterSecondInfoState,
+      gamesPlayed, enterThirdInfoState} = this.props;
+
+    //setting state for the start time of the game
+    if (this.state.startTime === 0) {
+      this.setState({startTime: Date.now()})
+    }
+
+    //Resets the state time of the game zero so the game repeats
+    const updateTime = () => {
+      this.setState({startTime: 0});
+    }
+
+    //Controls the top right page countdown clock
+    const mainTimerRenderer = (props) => {
+      if (props.total === 0) {
+        return (<Title gameState={false} replay={true}/>);
+      } else {
+        let milliseconds = props.milliseconds;
+        milliseconds = milliseconds / 100;
+        return (
+          <div>
+            <div className='timer right'>{props.seconds}.{milliseconds}</div>
```

Comment:
The timer is a great feature, but it's hard to see when it's very small in the top corner